pinot-query-runtime/src/test/resources/queries/ValueExpressions.json

```diff
@@ -0,0 +1,102 @@
+{
+  "subscript": {
+    "psql": "4.2.3",
+    "ignored": true,
+    "comment": "query runner test doesn't yet support multivalue columns",
+    "tables": {
+      "tbl": {
+        "schema":[{"name": "data", "type": "INT[]"}],
+        "inputs": [[[1]]]
+      }
+    },
+    "queries": [{
+      "description": "test array subscripting",
+      "sql": "SELECT data[0] FROM {tbl}"
+    }]
+  },
+  "scalar_function_call": {
+    "psql": "4.2.6",
+    "tables": {
+      "tbl": {
+        "schema":[{"name": "data", "type": "STRING"}],
+        "inputs": [[" foo "]]
+      }
+    },
+    "queries": [{
+      "description": "test basic function call syntax works",
+      "sql": "SELECT trim(data) FROM {tbl}"
+    }]
```

Comment:
i dont think this works b/c `data` seems to need escape in pinot SQL. it is explained into:
```
rel#10:LogicalProject.(input=LogicalTableScan#7,exprs=[TRIM(FLAG(BOTH), ' ', $0)])
```

Reply:
sorry not sure I understood?

Reply:
nope. my mistake. it is b/c the FLAG(BOTH) is actually a enum and it wasn't parsed correctly in `RexExpression.toDataType(RelDataType type)`